packages/ui/src/stories/formkit/Rating.stories.ts

```diff
@@ -0,0 +1,24 @@
+import { StoryFn } from '@storybook/vue3'
```

Comment:
Not sure what component is used here, but it shouldn't behave this way.

We should use vuesic rating instead.

<img width="152" alt="image" src="https://github.com/user-attachments/assets/f96ccdf2-42e0-4c62-a726-98c330ef56bf" />